### ISSUE TITLE
Fix security scan pipeline [retro action item]

### ISSUE THREAD

**Issue by jimmykarily** [kairos-io/kairos#2346]
so that we see the security warnings.

This story is about fixing the pipelines, not the warnings themselves.

**Comment by jimmykarily** [comment]
Github interface is not helping understand what the problem is. It seems that since we reported flavors, the old ones still appeared there. I just removed those and now trivy and grype scans look green.

There is another "scanning tool" which is linked to a still existing workflow and looks as if it fails. That workflow has probably changed and GitHub is confused. Deleting those doesn't work (https://github.com/kairos-io/kairos/security/code-scanning/tools/%20/status/configurations/actions-FZTWS5DIOVRC653POJVWM3DPO5ZS62LNMFTWKLTZMFWWY/a0a328ccd4fb71f4228ad8b36f6eb0bf8296e65412711565c236429ae3a34e9b).

I don't see what else we can do here. It's just badly implemented feature from GitHub, there are not actions we can take to fix it afaict.
